src/docs/truffle/reference/configuration.md

```diff
@@ -315,10 +315,11 @@ In the `compilers` object you can specify settings related to the compilers used
 Solidity compiler settings. Supports optimizer settings for `solc`, as well as other settings such as debug and metadata settings.
 
 You may specify...
-+ any solc-js version listed at [solc-bin](http://solc-bin.ethereum.org/bin/list.json). Specify the one you want and Truffle will get it for you.
-+ a natively compiled solc binary (you'll need to install this yourself, links to help below).
-+ a dockerized solc from one of images published [here](https://hub.docker.com/r/ethereum/solc/tags/).
++ any solc-js version (using semver) listed at [solc-bin](http://solc-bin.ethereum.org/bin/list.json). Specify the one you want and Truffle will get it for you.
++ "native" to use a natively compiled solc binary (you'll need to install this yourself, links to help below).
++ a dockerized solc tag from one of images published [here](https://hub.docker.com/r/ethereum/solc/tags/).
 + a path to a locally available solc
++ "pragma" to have Truffle autodetect solc versions from your source files. This can be used to compile using multiple versions of solc.
```

Comment:
This is not clear to me, perhaps we can offer a use case . A starting point...

> Use this option to resolve the appropriate compiler version for projects referencing modules with different solidity pragmas...

Reply:
There is probably another spot where it is more appropriate to go into more detail. This is not the place for what you are describing. Take a look at the descriptions for the other settings; they are all very brief.

Reply:
If we do that, it should probably be a separate PR.

Reply:
Is there a requirement to keep this section brief? I like the idea of a surfacing more information wherever it goes :)

Reply:
No I'm just saying rewriting this section to be something totally different seems like it should be a task different from the one I originally set out to accomplish.